lumi-checks/microbenchmarks/mpi/osu/osu_tests.py

```diff
@@ -0,0 +1,100 @@
+import os
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+from packaging import version
+from hpctestlib.microbenchmarks.mpi.osu import (osu_latency,
+                                                osu_bandwidth,
+                                                build_osu_benchmarks)
+
+
+cpu_build_variant = build_osu_benchmarks.get_variant_nums(
+    build_type='cpu'
+)
+
+stack_name    =  os.getenv('LUMI_STACK_NAME', None)
+stack_version =  os.getenv('LUMI_STACK_VERSION', None)
+
+environs = ['PrgEnv-gnu', 'PrgEnv-cray']
+
+if stack_name and stack_name == 'LUMI':
+    environs += ['cpeGNU', 'cpeCray']
+
+    if version.parse(stack_version) > version.parse('21.08'):
+        environs += ['cpeAOCC']
```

Comment:
I add to create this hacky solution in order to support the different software stack of LUMI. cpeAOCC has been introduced by LUMI/21.12 and is not present in 21.08. The PrgEnv-aocc provided by Cray is still broken.